pkg/apis/components/v1alpha1/types.go

```diff
@@ -37,6 +37,36 @@ type Component struct {
 	Scopes []string `json:"scopes,omitempty"`
 }
 
+// Kind returns the component kind.
+func (Component) Kind() string {
+	return "Component"
+}
+
+// +genclient
```

Comment:
Comments meant to be read by a machine should not have a space after `//`

Reply:
Thanks for catching this!

As you can [see](https://github.com/dapr/dapr/pull/5102/files/653dac8cb4caa5328d3b027ae9538a59258eb362#diff-b61d06975d6d1f5da12fdb8a6368b859979ae451e15d9c02825e91ea305f5b64R23) I'm actually using the same format as for Component CRD.

Reply:
Yes the convention throughout Go is that comments have a space after // and directives (meant to instruct a compiler or something else) have no space after //

Reply:
Fixed!